util/annotations/annotations.go

```diff
@@ -147,6 +147,7 @@ var (
 	PossibleNonCounterInfo                  = fmt.Errorf("%w: metric might not be a counter, name does not end in _total/_sum/_count/_bucket:", PromQLInfo)
 	HistogramQuantileForcedMonotonicityInfo = fmt.Errorf("%w: input to histogram_quantile needed to be fixed for monotonicity (see https://prometheus.io/docs/prometheus/latest/querying/functions/#histogram_quantile) for metric name", PromQLInfo)
 	IncompatibleTypesInBinOpInfo            = fmt.Errorf("%w: incompatible sample types encountered for binary operator", PromQLInfo)
+	MixedFloatsHistogramsInfo               = fmt.Errorf("%w: encountered a mix of histograms and floats for", PromQLInfo)
```

Comment:
See comment below, I think this should be something like this:

```suggestion
	HistogramIgnoredInAggregationInfo               = fmt.Errorf("%w: ignored histogram for", PromQLInfo)
```